src/RolloutScorer/RolloutScorer/Providers/RolloutScorerProvider.cs

```diff
@@ -462,8 +470,8 @@ public void SetupHttpClient(string azdoPat)
 
         public void SetupGithubClient(string githubPat)
```

Comment:
The GitHub client should be DIed where it's needed and this method should be refacorted out.